package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderV3.java

```diff
@@ -104,6 +104,7 @@ public void upload ( final File file, final String filename ) throws IOException
         }
         catch ( final IOException e )
         {
+            e.printStackTrace ( this.listener.error ( "Failed to create upload archive" ) );
```

Comment:
The `e.printStackTrace` print to the standard error that mean that is logger to the jenkins log and is not visible in the jenkins build console. Unless you login to the jenkins master node and find the (system or where you redirects output) logs you never see it. In general in Java application usually the `e.printStackTrace` use is discouraged infavor or any logger library (because could be configured the output)

`this.listener` in this case is the TaskListener that is a Jenkins wrapper for the log to print to the jenkins console log. So if your goal is advise the user why the build is failed than `this.listener` is the one that does your case. Maybe instead just notice "fail to create upload" if you post also the original cause message can be of more help in understanding what the problem is.